presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/BenchmarkSTIntersects.java

```diff
@@ -121,7 +121,7 @@ public void validateBenchmarkData()
         public void setup()
                 throws IOException
         {
-            simpleGeometry = stGeometryFromText(utf8Slice("POLYGON ((16.5 54, 16.5 54.1, 16.51 54.1, 16.8 54))"));
+            simpleGeometry = stGeometryFromText(utf8Slice("POLYGON ((16.5 54, 16.5 54.1, 16.51 54.1, 16.8 54, 16.5 54))"));
```

Comment:
> Make Polygon WKTs valid in test cases 

Although it is by the standard - it is a breaking change. Do we care that if somebody had existing WKT stored in the database that were compatible with ESRI it will stop working?

If we are fine with that, should we have a better release note?